htsget.md

```diff
@@ -27,6 +27,8 @@ Explicitly this API does NOT:
 
 All API invocations are made to a configurable HTTP(S) endpoint, receive URL-encoded query string parameters, and return JSON output. Successful requests result with HTTP status code 200 and have UTF8-encoded JSON in the response body, with the content-type `application/json`. The server may provide responses with chunked transfer encoding. The client and server may mutually negotiate HTTP/2 upgrade using the standard mechanism.
 
+The JSON response SHOULD be an object with the single key `htsget`.  The value is an object as described in the [Response JSON fields](#response-json-fields) and [Error Response JSON fields](#error-response-json-fields) sections.  This ensures that, apart from whitespace differences, the message always starts with the same prefix.  The presence of this prefix can be used as part of a client's response validation.
```

Comment:
The RFC 2119 shouty verbs are for quantifying implementation behaviour, while this section is probably clearest if it discusses the protocol as a Platonic ideal. The second sentence can be read as saying that the value attached to the `htsget` key is what is described in the tables, but the tables describe the whole thing and include the object containing the `htsget` key/value itself too.

So perhaps this would be clearer as “The JSON response is an object with the single key `htsget`, as described in the […]”.